static/switch-btn.js

```
@@ -4,49 +4,56 @@ var switchButton = document.querySelector('.switch-button');
 var switchBtnLeft = document.querySelector('.switch-button-case.left');
 var activeSwitch = document.querySelector('.active');
-var numDraws = document.getElementById('numDraws');//決定要抽幾抽的表單
 
-//抓轉蛋機外觀元素，用來調整轉蛋機的外觀
+// 決定要抽幾抽的表單
+var numDraws = document.getElementById('numDraws');
+
+// 抓轉蛋機外觀元素，用來調整轉蛋機的外觀
 var st3 = document.getElementsByClassName('st3');
 var st4 = document.getElementsByClassName('st4');
 var st5 = document.getElementsByClassName('st5');
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)